### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by LeoDiduch**

### CODE REVIEW

**Review by BastienSaulnier — APPROVED** [review]
SnE

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
 
 
 function App() {
   document.title = siteTitle;
+  const location = useLocation();
 
   return (
     <div className="App">
-      <NavBar />
+      {location.pathname.includes("admin") ? <NavBarBO /> : <NavBar />}
```
Comment (LoicBrassart) [inline frontend/src/App.js:28]: #notBlocking - The idea is good, but it's a security breach: nothing forbids your user from typing "admin" in the url (`/admin` was for a very long time the default route to administration for WP websites... Wannabe hackers may try and type this at "random")
I'd at least sort my admin routes as `PrivateRoute` (see `old_roleet` for that), to limit what attackers could access through this

File: frontend/src/pages/AmbassadorsPageBO.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+export default function AmbassadorsPageBO() {
+  return <div></div>;
```
Comment (LoicBrassart) [inline frontend/src/pages/AmbassadorsPageBO.jsx:4]: These pages are a bit empty for now :p 
I'd have created the AdminNavbar but not added the links for now

**Review by LoicBrassart — APPROVED** [review]
Saw no real Evil

**Pull request merged**